spec-main/api-native-theme-spec.ts

```diff
@@ -8,6 +10,58 @@ describe('nativeTheme module', () => {
     })
   })
 
+  describe('nativeTheme.themeSource', () => {
+    afterEach(() => {
+      nativeTheme.themeSource = 'system'
+    })
+
+    it('is system by default', () => {
+      expect(nativeTheme.themeSource).to.equal('system')
+    })
+
+    it('should override the value of shouldUseDarkColors', () => {
+      nativeTheme.themeSource = 'dark'
+      expect(nativeTheme.shouldUseDarkColors).to.equal(true)
+      nativeTheme.themeSource = 'light'
+      expect(nativeTheme.shouldUseDarkColors).to.equal(false)
+    })
+
+    it('should emit the "updated" event when it is set and the resulting "shouldUseDarkColors" value changes', () => {
+      nativeTheme.themeSource = 'dark'
+      let called = false
+      nativeTheme.once('updated', () => {
+        called = true
+      })
+      nativeTheme.themeSource = 'light'
+      expect(called).to.equal(true)
+    })
+
+    it('should not emit the "updated" event when it is set and the resulting "shouldUseDarkColors" value is the same', () => {
+      nativeTheme.themeSource = 'dark'
+      let called = false
+      nativeTheme.once('updated', () => {
+        called = true
+      })
+      nativeTheme.themeSource = 'dark'
+      expect(called).to.equal(false)
+    })
+
+    describe('on macOS 10.14', () => {
```

Comment:
prefer `ifdescribe`:
```suggestion
    ifdescribe(process.platform === 'darwin' && semver.ge(os.release(), '18.0.0'))('on macOS 10.14', () => {
```